cli/actions/setup.js

```diff
@@ -86,6 +86,9 @@ module.exports = (config, flags, args) => {
       commands = uniq(commands.concat(tempScripts));
     }
 
+    // This is the default test script added by 'npm init'.
+    const npmInitDefaultTestScript = 'echo \"Error: no test specified\" && exit 1'; // eslint-disable-line no-useless-escape
```

Comment:
@aterranova-bv it escapes it for json--when you read it into node it will just be a string without those escapes.

```
test 🙀  > cat package.json
{
  "name": "test",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC"
}
test 🙀  > node
> const pkg = require('./package.json')
undefined
> pkg.scripts.test
'echo "Error: no test specified" && exit 1'
```